### DISCUSSION TITLE
wompi y addi no caragan simultaneamente

### DISCUSSION THREAD

**Discussion by jose5789** [woocommerce/woocommerce#46285] [Checkout Flow]
hola con la última actualización de woocommerce no me aparecian los métodos de pago creados y activos cuando me paso esto cambien el modelo de bloques de checkout por el clasico pero no resolvio el problema aun sigue apareciendo un solo método activo de 3 disponibles

**Comment by senadir** [comment]
Hey! What payment methods do you have?
